app/uk.gov.hmrc.helptosavestridefrontend/connectors/HelpToSaveConnector.scala

```diff
@@ -150,6 +159,29 @@ class HelpToSaveConnectorImpl @Inject() (http:                              WSHt
           }
       })
 
+  override def createAccount(nSIUserInfo: NSIUserInfo)(implicit hc: HeaderCarrier, ec: ExecutionContext): Result[CreateAccountResult] = {
+
+    EitherT[Future, String, CreateAccountResult](http.post(createAccountUrl, nSIUserInfo).map[Either[String, CreateAccountResult]] { response ⇒
+      response.status match {
+        case Status.CREATED ⇒
+          logger.debug("createAccount returned 201 (Created)")
```

Comment:
it might be handy to modify `LoggingOps` in `Logging` to be able to pass in a nino parameter at debug level